ServerSocketTest/src/main/Main.java

```diff
@@ -50,38 +74,123 @@ public static void main(String[] args) {
 				
 				
 				//get the first line of the request
-				String firstLine=request.toString().split("\n")[0];
+				String firstLine=header.toString().split("\n")[0];
 				//System.out.println(firstLine);
 				//get the second element from first line
 				String resource=firstLine.split(" ")[1];
```

Comment:
I'm confused where the value of resource comes from I see it comes from the firstLine string but how does it get the value for "people".

Reply:
When we make a GET request from curl ( enter the following into command line: curl localhost:8097/people ) we get the following:
GET /people HTTP/1.1 <----- there is a hidden \n here
Host: localhost:8097
User-Agent: curl/8.4.0
Accept: */*

line 77 pulls out the first line of the header which is:
GET /people HTTP/1.1
line 80 splits everywhere we see a space which results in:
GET
/people
HTTP/1.1

and specifying [1] gets the second element from the split which I call the resource